### ISSUE TITLE
Fatal error

### ISSUE THREAD

**Issue by MonMon3** [nova-framework/framework#834]
Warning: set_exception_handler() expects the argument (Core\Logger::ExceptionHandler) to be a valid callback in /var/www/html/test/system/bootstrap.php on line 18

Warning: set_error_handler() expects the argument (Core\Logger::ErrorHandler) to be a valid callback in /var/www/html/test/system/bootstrap.php on line 19

Fatal error: Class 'Core\Config' not found in /var/www/html/test/app/Config.php on line 114

**Comment by pandory-network** [comment]
Have u run composer install ?

**Comment by MonMon3** [comment]
No, copying my archive.
Installing by compose give me...
Warning: file_put_contents(errorlog.html): failed to open stream: Permission denied in /var/www/html/LMNZPhoneLog/app/Core/Logger.php on line 158

An error occured, The error has been reported.

**Comment by LuckyCyborg** [comment]
You have file permission problems.

And, again, why you install the framework in a sub-directory, while it is designed to live in its virtual host or dedicated site?

**Comment by MonMon3** [comment]
Have put in root.
chmod -R 777 *
same thing.

**Comment by geomorillo** [comment]
We should stop support if installed in sub-directory :=)

**Comment by MonMon3** [comment]
I have put in root....now....

**Comment by geomorillo** [comment]
"while it is designed to live in its virtual host or dedicated site"....  root != virtual host :)

**Comment by MonMon3** [comment]
Let me rephrase that. It lives in /var/www/html/.... by root I mean that.
On 18/05/2016 5:10 pm, "Jhobanny Morillo" <notifications@github.com> wrote:
>
> "while it is designed to live in its virtual host or dedicated site"....
root != virtual host :)

**Comment by LuckyCyborg** [comment]
@MonMon3 First of all you are NOT in a dedicated Virtual Host or website. You are in **http://localhost/Test/**, see:

/var/www/html/**Test**/app/Core/Logger.php 

Secondly, you do not have a clue how to set the permissions under Linux. The paths should be accessible for Apache, then usually you have to do into **/var/www/html**
```
chown -R apache:apache .
```

Secondly, please learn how to do and install Nova Framework in a **Virtual Host**. If you aren't able to figure how to configure that, you skills are not high enough for using our Framework.

Sorry, but a MVC Framework is not exactly the best thing to start to learn PHP.

**Comment by LuckyCyborg** [comment]
@geomorillo 

> We should stop support if installed in sub-directory :=)

Definitively! I hope in one day @daveismyname will understand that suggesting to end-users the sub-directory installation do **NO** favor to them, even with dozen videos on subject. :smirk:

**Comment by MonMon3** [comment]
Oh my god... I said I changed it........

Not using apache, using httpd.

I've used simple mvc before and it worked perfectly using it in a dir.

And I did install it using command line....

**Comment by LuckyCyborg** [comment]
@MonMon3 Please configure a Virtual Host called **novaframework.dev** and install there the Nova Framework with the document root pointing to **/public** directory.

**Comment by geomorillo** [comment]
"I've used simple mvc before and it worked perfectly using it in a dir."
Sorry to say this but v3 is different than v2 it wont work the same. Is a different beast.

**Comment by MonMon3** [comment]
Thanks guys for the help. Will try when I next have access to server.

**Comment by LuckyCyborg** [comment]
So we can close this Issue?

**Comment by MonMon3** [comment]
Ummmm if you really want to. ...
